src/callback_handler/oci.rs

```diff
@@ -15,9 +16,16 @@ impl Client {
 
     /// Fetch the manifest digest of the OCI resource referenced via `image`
     pub async fn digest(&self, image: &str) -> Result<String> {
-        let image_with_proto = format!("registry://{}", image);
-        self.registry
+        // this is needed to expand names as `busybox` into
+        // fully resolved references like `docker.io/library/busybox`
+        let image_ref: Reference = image.parse()?;
+
+        let image_with_proto = format!("registry://{}", image_ref.whole());
+        let image_digest = self
+            .registry
             .manifest_digest(&image_with_proto, self.sources.as_ref())
-            .await
+            .await?;
+        serde_json::to_string(&image_digest)
```

Comment:
why do we need this? `manifest_digest` returns a `Result<String>`. Can we just return `manifest_digest` ?